.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+# DDCE-(StoryNumberHere)
+
```

Comment:
Nitpick, it's just wording changes that I've been trying to get into the templates for the PR reviewer section - https://github.com/hmrc/ras-frontend/blob/master/.github/PULL_REQUEST_TEMPLATE.md

Reply:
on it. also adding 'dist: trusty' line to travis.yml while at it - found that's the reason why it fails on trying to get oraclejdk8.